.github/workflows/codeql.yaml

```diff
@@ -1,28 +1,39 @@
 name: "CodeQL"
 on:
-  workflow_dispatch:
   push:
-    branches: [ master ]
+    branches:
+    - master
+    - release/v1.4
   pull_request:
-    branches: [ master ]
+    branches: 
+    - master
+    - release/v1.4
```

Comment:
Is this just for release/1.4 branch and not for other release branches going forward ?